src/Core/Users/user.php

```diff
@@ -0,0 +1,142 @@
+<?php
+namespace Meetup\Login\Core;
+class User implements UserInterface {
+    protected $firstName;
+    protected $lastName;
+    protected $emailid;
+    protected $created;
+    protected $datetime;
+    protected $passwd;
+    /**
+     * Set first name
```

Comment:
Annotations not required in classes, as we are documenting interfaces. Please remove them.